### PR TITLE
feat: Add LIDO-ORACLE-LOW-BALANCE alert

### PR DESCRIPTION
**Pull request by dgusakov**
Emitted when oracle balance gets below 0.3 ETH.
Will be repeated every 3 days

### CODE REVIEW

File: ethereum/src/agent-lido-oracle.ts [hunk 1]
```diff
@@ -271,8 +256,91 @@ export async function handleBlock(blockEvent: BlockEvent) {
...
+  oracle: string,
+  blockEvent: BlockEvent,
+  findings: Finding[]
+) {
+  const now = blockEvent.block.timestamp;
+  const lastAlert = oraclesBalanceLastAlert.get(oracle) || 0;
+  if (now > lastAlert + THREE_DAYS) {
+    const balance = new BigNumber(
+      String(await ethersProvider.getBalance(oracle))
+    ).div(ETH_DECIMALS);
```
Comment (avsetsin) [inline ethereum/src/agent-lido-oracle.ts:324]: looks like `await ethersProvider.getBalance(oracle)` already returns BigNumber

**Review by avsetsin — APPROVED** [review]
lgtm

File: ethereum/src/agent-lido-oracle.ts [hunk 2]
```diff
@@ -43,30 +45,40 @@ const ZERO = new BigNumber(0);
...
+export const name = "LidoOracle";
+
+async function getOracles() {
+  const lidoOracle = new ethers.Contract(
+    LIDO_ORACLE_ADDRESS,
+    LIDO_ORACLE_ABI,
+    ethersProvider
+  );
+
+  return String(await lidoOracle.functions.getOracleMembers()).split(",");
```
[the remark this answers is not in the document]
Comment (dgusakov) [reply ethereum/src/agent-lido-oracle.ts:67]: Nope. It returns a string with comma separated values

File: ethereum/src/agent-lido-oracle.ts (hunk 1 above)
Comment (dgusakov) [reply ethereum/src/agent-lido-oracle.ts:324]: Yep. But there are several implementations of the BigNumber and ethers returns the wrong one

**Review by vgorkavenko — APPROVED** [review]

**Pull request merged**